geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/RasterRDD.scala

```diff
@@ -56,6 +57,32 @@ abstract class TileRDD[K: ClassTag] {
   def rdd: RDD[(K, MultibandTile)]
   def keyClass: Class[_] = classTag[K].runtimeClass
   def keyClassName: String = keyClass.getName
+
+  def reclassify(
+    intMap: java.util.Map[Int, Int]
+  ): TileRDD[_] = {
+    val scalaMap = intMap.asScala.toMap
+
+    reclassify(scalaMap.keySet, BreakMap(scalaMap))
+  }
+
+  def reclassifyDouble(
+    doubleMap: java.util.Map[Double, Double]
+  ): TileRDD[_] = {
```

Comment:
This doesn't really work for doubles since you can't enumerate a floating point range to perform the substitution here. You really need a concept of a range with a starting and ending point.